front/src/components/UserForm.tsx

```diff
@@ -50,36 +50,36 @@ const UserFormDesign: React.FC = () => {
        const [success, setSuccess] = useState(false);
        const [error, setError] = useState<string | null>(null);
 
-       const onSubmit = async (data: FormValues) => {
-              setLoading(true);
-              setError(null);
-              setSuccess(false);
-
-
-              fetch("/api/post/create",
-                     {
-                            method: "POST",
-                            headers: { "Content-Type": "application/json" },
-                            body: JSON.stringify(data)
-                     }
-              ).then((result) => {
-                     if (result?.ok) {
-                            setSuccess(true);
-                     }
-                     else {
-                            throw new Error();
-                     }
-
-              }).
-                     catch((e) => {
-                            setSuccess(false);
-                            setError("Submission failed");
-                     }).finally(() => {
-                            setLoading(false);
-                            reset();
-                     })
-       };
-
+       
+const onSubmit = async (data: FormValues) => {
+       setLoading(true);
+       setError(null);
+       setSuccess(false);
+   
+       let resetFields: Partial<FormValues> = {}; // Define which fields to reset
+   
+       try {
+           const result = await fetch("/api/post/create", {
+               method: "POST",
+               headers: { "Content-Type": "application/json" },
+               body: JSON.stringify(data),
+           });
+   
+           if (result.ok) {
+               setSuccess(true);
+               resetFields = { ...data, Age:"" ,}; // Reset all fields except 'Age'
+           } else {
+               throw new Error();
+           }
+       } catch (e) {
+           setSuccess(false);
+           setError("Submission failed");
+       } finally {
+           setLoading(false);
+           reset(resetFields); // Reset form fields conditionally
```

Comment:
`reset(resetFields); // Reset form fields conditionally`
not required set back to `reset();`